### ISSUE TITLE
"Windows protected your computer" during installation

### ISSUE THREAD

**Issue by aubergine10** [desktop/desktop#8901]
### Describe the bug

Windows 10 latest build shows protection warning screen whenever I try installing desktop after downloading from https://desktop.github.com/

### Version & OS

GitHub Desktop Version: 2.2.4

Winver: Windows 10 Home, 1909 with latest updates, OS Build 18363.418

### Steps to reproduce the behavior

1. Download the .exe from https://desktop.github.com/
2. Try opening it from the browser download panel
3. Windows moans about unknown publisher

### Expected behavior

Would expect the download to be digitally signed to avoid that warning.

### Actual behavior

Windows moans about unknown publisher.

### Screenshots

![image](https://user-images.githubusercontent.com/1386719/72209840-eb00bc00-34aa-11ea-8cc3-d76587306330.png)

**Comment by niik** [comment]
Hi @aubergine10, thanks for opening this and thanks especially for including so much detail about your system, it really helps.

I've attempted to reproduce this with a newly installed VM upgraded to 1909 with all other available patches and I've been unsuccessful. Do you have another system that you could try to see if it behaves the same?

> Would expect the download to be digitally signed to avoid that warning.

The installer is signed which makes this quite confusing. I initially suspected that this could have something to do with us using SHA-1 as the file digest algorithm and subsequently opened #8906 to address that but since I can't reproduce it it's hard for me to test. Could you be so kind and right click on the downloaded file, and take a screenshot of the signatures tab, like this:

<img width="392" alt="image" src="https://user-images.githubusercontent.com/634063/72239612-d410b580-35e1-11ea-9897-02b1aafc2606.png">

**Comment by aubergine10** [comment]
> Do you have another system that you could try to see if it behaves the same?

Yup, I have a differnet system I can test on (very different, a LattePanda SBC with Intel Atom CPU lol). I'll do some testing on that tomorrow.

One other thing I forgot to mention is that I downloaded via Firefox with DNS over HTTPS (DOH) feature enabled. Shouldn't make a difference but thought worth mentioning just in case.

> Could you be so kind and right click on the downloaded file, and take a screenshot of the signatures tab

I'd deleted the original installer so just downloaded it again:

![image](https://user-images.githubusercontent.com/1386719/72297704-5b970c80-3654-11ea-98e8-f8083f8a108b.png)

Also, I tried using this newly downloaded installer and no warning from Windows this time so not sure what is happening. Maybe transient Windows glitch?
